collectors/gsuite/README.md

```diff
@@ -10,35 +10,61 @@ forward logs to the Alert Logic CloudInsight backend services.
 
 # Installation
 
-Refer to [CF template readme](./cfn/README.md) for installation instructions.
+### 1. G Suite Setup
+
+1. Create a Custom admin role. [Link](https://support.google.com/a/answer/2406043)
+2. Create a custom admin role with the 'Reports' privilege assigned
```

Comment:
Step 1 is no longer necessary, could you remove that? Make sure to add the link to step 2